mix.exs

```diff
@@ -15,7 +15,7 @@ defmodule Guardian.Mixfile do
       name: "Guardian",
       app: :guardian,
       version: @version,
-      elixir: "~> 1.1",
+      elixir: "~> 1.2",
```

Comment:
After conferring with @hassox and @scrogson, we're going to bump this to 1.3 and drop 1.2